app/models/form/lettings/questions/location_id_search.rb

```diff
@@ -2,10 +2,8 @@ class Form::Lettings::Questions::LocationIdSearch < ::Form::Question
   def initialize(id, hsh, page)
     super
     @id = "location_id"
-    @check_answer_label = "Location"
-    @header = header_text
-    @hint_text = '<div class="govuk-inset-text">This scheme has 20 or more locations.</div>Enter postcode or address.'
     @type = "select"
+    @hint = I18n.t("forms.#{form.start_date.year}.#{copy_key}.search_hint_text", default: "") if form.start_date.present?
```

Comment:
Should this be @hint_text?, also, wouldn't it just get pulled as long as the copy key is correct and the key is hint_text instead of search_hint_text?

Reply:
There are two questions with the same id "location_id", there's the default which displays the answer options as radio buttons and then a few months ago I add an alternative which displayed the options in a search box (when there were more than 20 locations). The default hint_text is for the radio buttons version and this search_hint_text is for the alternative option.

Reply:
Back when adding the search version, I kept the id's of the two different questions the same so they displayed the same way in the url.

Reply:
Ah I see, I've been duplicating and setting relevant copy keys when there are 2 versions of the question, in case we want to update the wording for other attributes in the future, so I'd have something like
```
          location_id:
            over_20_results:
              page_header: "Location"
              check_answer_label: "Location"
              hint_text:  '<div class="govuk-inset-text">This scheme has 20 or more locations.</div>Enter postcode or address.'
              question_text: "Which location is this log for?"
             under_20_results:
               page_header: "Location"
              check_answer_label: "Location"
              hint_text:  '<div class="govuk-inset-text">This scheme has 20 or more locations.</div>Enter postcode or address.'
              question_text: "Which location is this log for?"
 ```
 
This also would meant that we don't have to define a new/custom key, but idm whether you change it or keep it as is

Reply:
Ah, this is a good idea I'll change it to something like this to keep this consistent.